File: CPP/aksesoris.cpp
#include <iostream>
#include <string>
#include "petshop.cpp"

using namespace std;

class aksesoris:public petshop{
    private: 
        string jenis,bahan,warna;
    public:
        aksesoris(){
            this->jenis="";
            this->bahan="";
            this->warna="";
        }
        aksesoris(string jenis,string bahan,string warna){
            this->jenis=jenis;
            this->bahan=bahan;
            this->warna=warna;
        }
        
        string get_jenis(){
            return this->jenis;
        }
        void set_jenis(string jenis){
            this->jenis=jenis;
        }
        
        string get_bahan(){
            return this->bahan;
        }
        void set_bahan(string bahan){
            this->bahan=bahan;
        }

        string get_warna(){
            return this->warna;
        }
        void set_warna(string warna){
            this->warna=warna;
        }

        ~aksesoris(){
        }
};

File: PYTHON/Aksesoris.py
from Petshop import Petshop  # Mengimpor kelas Petshop dari file petshop.py

class Aksesoris(Petshop):
    def __init__(self, jenis="", bahan="", warna="", id=0, harga_produk=0, stok_produk=0, nama_produk=""):
        # Memanggil constructor kelas induk (Petshop) secara manual
        Petshop.__init__(self, id, harga_produk, stok_produk, nama_produk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def get_jenis(self):
        return self.jenis

    def set_jenis(self, jenis):
        self.jenis = jenis

    def get_bahan(self):
        return self.bahan

    def set_bahan(self, bahan):
        self.bahan = bahan

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna
